cli/src/com/redhat/ceylon/common/tools/AbstractTestTool.java

```diff
@@ -92,9 +92,11 @@ public void setVersion(String version) {
         this.version = version;
     }
     
-    @Option(longName = "tap")
-    @Description("Enables the Test Anything Protocol v13.")
-    public void setTap(boolean tap) {
+    @OptionArgument(argumentName = "file")
```

Comment:
Well, the realistic answer is that probably no one’s using this feature at all right now, so there’s little reason to add this weird special case. And you’re right, in the Ant task printing to stdout is next to useless, because you’ll have to carefully extract the TAP report from the rest of the Ant output.